Tests/SolanaTests/Models/TransactionTest.swift

```diff
@@ -0,0 +1,194 @@
+//
+//  TransactionTest.swift
+//  
+//
+//  Created by Mirek Rousal on 25/10/2022.
+//
+
+import Foundation
+import XCTest
+
+@testable import Solana
+
+final class TransactionTest: XCTestCase {
+    
+    @available(iOS 13.0.0, *)
```

Comment:
Similar comment here about the availability being language related, not OS related

Reply:
Moved to the class level.